src/main/java/controller/Controller.java

```diff
@@ -0,0 +1,72 @@
+package controller;
+
+import domain.CarFactory;
+import domain.Cars;
+import domain.RacingCount;
+import domain.RacingRule;
+import dto.CarStatus;
+import view.InputView;
+import view.OutputView;
+
+import java.util.List;
+
+public class Controller {
+
+    private final InputView inputView;
+    private final OutputView outputView;
+
+    public Controller(final InputView inputView, final OutputView outputView) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+    }
+
+    public void run() {
+        final Cars cars = initCars();
+        final RacingCount racingCount = inputRacingCount();
+
+        final List<String> winners = racing(cars, racingCount);
+
+        outputView.printWinners(winners);
+    }
+
+    private Cars initCars() {
+        try {
+            final List<String> carNames = inputView.inputCars();
+
+            return CarFactory.generateCars(carNames);
+        } catch (IllegalArgumentException e) {
+            outputView.printInputCarNamesErrorMessage();
+
+            return initCars();
+        }
+    }
+
+    private RacingCount inputRacingCount() {
+        try {
+            return inputView.inputRacingCount();
+        } catch (IllegalArgumentException e) {
+            outputView.printInputRacingCountErrorMessage();
+
+            return inputRacingCount();
+        }
+    }
+
+    private List<String> racing(final Cars cars, final RacingCount racingCount) {
+        outputView.printResultMessageTitle();
+        for (int i = 0; i < racingCount.getCount(); i++) {
+            final List<CarStatus> raceResult = cars.race();
+            outputView.printRacingResult(raceResult);
+        }
+
+        return getWinners(cars);
+    }
+
+    private List<String> getWinners(final Cars cars) {
+        final RacingRule racingRule = new RacingRule();
+
+        return racingRule.getWinners(cars)
+                .stream()
+                .map(car -> car.getStatus().name())
```

Comment:
```suggestion
                .map(car -> car.getName())
```

[의견] 어떤 표현을 할 때 되도록 객체 구조를 보이지 않게 하는 것이 가독성을 개선할 수 있을 것 같아요.

Reply:
`getter 사용을 지양하라`
라는 말을 의식하다보니 어거지로 더 이상한 방법을 사용하게 되었네요.
객체 구조를 외부에 노출하지 않도록 코드를 바꿔볼게요.
👍